pages/photo-exhibitions/index.js

```diff
@@ -0,0 +1,255 @@
+import React from 'react';
+
+import Head from 'next/head';
+import GridList from '@material-ui/core/GridList';
+import GridListTile from '@material-ui/core/GridListTile';
+import { makeStyles, useTheme  } from '@material-ui/core/styles';
+
+import { getMenu } from '../../lib/api';
+
+import Navigation from '../../components/Navigation';
+import TopNavBar from '../../components/TopNavbar';
+
+import Box from "@material-ui/core/Box";
+import Typography from "@material-ui/core/Typography";
+import Grow from '@material-ui/core/Grow';
+
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+
+import {
+  FiCard,
+  FiCardActionArea,
+  FiCardActions,
+  FiCardContent,
+  FiCardMedia
+} from "../../components/ImageCard";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    [theme.breakpoints.down("769")]: {
+      marginLeft: '0% !important',
+      width: '100%',
+    },
+    [theme.breakpoints.down("1025")]: {
+      width: '100%',
+    },
+    [theme.breakpoints.up("1100")]: {
+      marginLeft: '10.5%',
+      width: '89.5%',
+    },
+    display: 'flex',
+    flexGrow: 1,
+    marginTop: 100,
+    
+    flexWrap: 'wrap',
+    justifyContent: 'space-around',
+    overflow: 'hidden',
+    alignContent: 'center',
+    alignItems: 'center',
+    verticalAlign: "middle",
+  },
+  rotatingImages: {
+    margin: 'auto',
+  },
+  topBanner: {
+    height: '100%',
+  },
+  bannerDiv: {
+    flexGrow: 1,
+    [theme.breakpoints.down("sm")]: {
+      height: 215,
+      marginBottom: 86, 
+    },
+    [theme.breakpoints.up("1024")]: {
+      height: 215,
+      marginBottom: 86, 
+      marginLeft: 200,
+    },
+    [theme.breakpoints.up("lg")]: {
+      height: 215,
+      marginBottom: 86, 
+    },
+    [theme.breakpoints.up("xl")]: {
```

Comment:
this breakpoint shouldnt be necessary since `theme.breakpoints.up('lg')` should hit large and up which includes xl

Reply:
understandable, but they do have effect for smaller screens i.e if on large monitor and bring up the control panel then responsive for that size too which I think is neat.